lib/rspec/rails/example/rails_example_group.rb

```diff
@@ -2,11 +2,6 @@
 # suite and ammeter.
 require 'rspec/rails/matchers'
 
-if ::Rails::VERSION::MAJOR >= 7
-  require 'active_support/current_attributes/test_helper'
-  require 'active_support/execution_context/test_helper'
-end
```

Comment:
Why are we doing this? Is there harm in having these always required?

Reply:
I will update these soon, thanks. There shouldn't be any harm. I just went with one approach here as I was just starting out without having an understanding from rspec's perspective at the time.

Reply:
This was updated so the require statements are back in the original location with no diff and just the include statements later in the file. Thanks!